building as `20220331T130939`

build successful: `03-31 11:44:49.190 20220331T130939: digest: sha256:3326add76cf762a06c80d51daadf589878b2fd33d7eb132f3418a1711f20f83f size: 5967`

test result: failure
```
[vcs 2022-03-31T19:02:11.281Z] adding file changes
[vcs 2022-03-31T19:02:11.779Z] added 4 changesets with 5 changes to 4 files (+1 heads)
[vcs 2022-03-31T19:02:11.780Z] new changesets 66b5f8936fa8:59ec5e25b59b (4 drafts)
[vcs 2022-03-31T19:02:12.087Z] (setting sparse config to profile build/sparse-profiles/perftest)
[vcs 2022-03-31T19:02:12.089Z] ** unknown exception encountered, please report by visiting
[vcs 2022-03-31T19:02:12.089Z] ** https://mercurial-scm.org/wiki/BugTracker
[vcs 2022-03-31T19:02:12.089Z] ** Python 3.6.9 (default, Dec  8 2021, 21:08:43) [GCC 8.4.0]
[vcs 2022-03-31T19:02:12.089Z] ** Mercurial Distributed SCM (version 5.9.3)
[vcs 2022-03-31T19:02:12.089Z] ** Extensions loaded: histedit, purge, rebase, robustcheckout, share, sparse
[vcs 2022-03-31T19:02:12.089Z] Traceback (most recent call last):
[vcs 2022-03-31T19:02:12.089Z]   File "/usr/local/bin/hg", line 61, in <module>
[vcs 2022-03-31T19:02:12.089Z]     dispatch.run()
[vcs 2022-03-31T19:02:12.089Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 144, in run
[vcs 2022-03-31T19:02:12.089Z]     status = dispatch(req)
[vcs 2022-03-31T19:02:12.089Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 250, in dispatch
[vcs 2022-03-31T19:02:12.089Z]     status = _rundispatch(req)
[vcs 2022-03-31T19:02:12.089Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 294, in _rundispatch
[vcs 2022-03-31T19:02:12.089Z]     ret = _runcatch(req) or 0
[vcs 2022-03-31T19:02:12.089Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 470, in _runcatch
[vcs 2022-03-31T19:02:12.089Z]     return _callcatch(ui, _runcatchfunc)
[vcs 2022-03-31T19:02:12.089Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 480, in _callcatch
[vcs 2022-03-31T19:02:12.089Z]     return scmutil.callcatch(ui, func)
[vcs 2022-03-31T19:02:12.090Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/scmutil.py", line 153, in callcatch
[vcs 2022-03-31T19:02:12.090Z]     return func()
[vcs 2022-03-31T19:02:12.090Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 460, in _runcatchfunc
[vcs 2022-03-31T19:02:12.090Z]     return _dispatch(req)
[vcs 2022-03-31T19:02:12.090Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 1274, in _dispatch
[vcs 2022-03-31T19:02:12.090Z]     lui, repo, cmd, fullargs, ui, options, d, cmdpats, cmdoptions
[vcs 2022-03-31T19:02:12.090Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 918, in runcommand
[vcs 2022-03-31T19:02:12.090Z]     ret = _runcommand(ui, options, cmd, d)
[vcs 2022-03-31T19:02:12.090Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 1285, in _runcommand
[vcs 2022-03-31T19:02:12.090Z]     return cmdfunc()
[vcs 2022-03-31T19:02:12.090Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/dispatch.py", line 1271, in <lambda>
[vcs 2022-03-31T19:02:12.090Z]     d = lambda: util.checksignature(func)(ui, *args, **strcmdopt)
[vcs 2022-03-31T19:02:12.091Z]   File "/usr/local/lib/python3.6/dist-packages/mercurial/util.py", line 1886, in check
[vcs 2022-03-31T19:02:12.091Z]     return func(*args, **kwargs)
[vcs 2022-03-31T19:02:12.091Z]   File "/usr/local/src/robustcheckout.py", line 211, in robustcheckout
[vcs 2022-03-31T19:02:12.091Z]     noupdate=noupdate,
[vcs 2022-03-31T19:02:12.091Z]   File "/usr/local/src/robustcheckout.py", line 786, in _docheckout
[vcs 2022-03-31T19:02:12.091Z]     timeit("sparse_update_config", "sparse-update-config"),
[vcs 2022-03-31T19:02:12.091Z] AttributeError: __enter__
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -87,7 +87,6 @@ ADD https://github.com/taskcluster/taskcluster/releases/download/v${TC_VERSION}/
 ADD https://github.com/taskcluster/taskcluster/releases/download/v${TC_VERSION}/livelog-linux-amd64 /usr/local/bin/livelog
 ADD https://github.com/taskcluster/taskcluster/releases/download/v${TC_VERSION}/taskcluster-proxy-linux-amd64 /usr/local/bin/taskcluster-proxy
 ADD https://github.com/taskcluster/taskcluster/releases/download/v${TC_VERSION}/start-worker-linux-amd64 /usr/local/bin/start-worker
-#ADD https://hg.mozilla.org/hgcustom/version-control-tools/raw-file/tip/hgext/robustcheckout/__init__.py /usr/local/src/robustcheckout.py
 ADD https://hg.mozilla.org/mozilla-central/raw-file/260e22f03e984e0ced16b6c5ff63201cdef0a1f6/testing/mozharness/external_tools/robustcheckout.py /usr/local/src/robustcheckout.py
 
 # for testing builds (these lines mirror above), copy above artifacts from the downloads dir

--- a/Dockerfile
+++ b/Dockerfile
@@ -93,7 +93,7 @@ ADD https://hg.mozilla.org/mozilla-central/raw-file/260e22f03e984e0ced16b6c5ff63
 # COPY downloads/node-v8.11.3-linux-x64.tar.gz /builds/worker/Downloads
 # COPY downloads/android-sdk_r24.3.4-linux.tgz /builds/worker/Downloads
 # COPY downloads/sdk-tools-linux-4333796.zip /builds/worker/Downloads
-# COPY downloads/generic-worker-nativeEngine-linux-amd64 /usr/local/bin/generic-worker
+# COPY downloads/generic-worker-simple-linux-amd64 /usr/local/bin/generic-worker
 # COPY downloads/livelog-linux-amd64 /usr/local/bin/livelog
 # COPY downloads/taskcluster-proxy-linux-amd64 /usr/local/bin/taskcluster-proxy
 # COPY downloads/start-worker-linux-amd64 /usr/local/bin/start-worker

--- a/Dockerfile
+++ b/Dockerfile
@@ -87,7 +87,8 @@ ADD https://github.com/taskcluster/taskcluster/releases/download/v${TC_VERSION}/
 ADD https://github.com/taskcluster/taskcluster/releases/download/v${TC_VERSION}/livelog-linux-amd64 /usr/local/bin/livelog
 ADD https://github.com/taskcluster/taskcluster/releases/download/v${TC_VERSION}/taskcluster-proxy-linux-amd64 /usr/local/bin/taskcluster-proxy
 ADD https://github.com/taskcluster/taskcluster/releases/download/v${TC_VERSION}/start-worker-linux-amd64 /usr/local/bin/start-worker
-ADD https://hg.mozilla.org/hgcustom/version-control-tools/raw-file/tip/hgext/robustcheckout/__init__.py /usr/local/src/robustcheckout.py
+#ADD https://hg.mozilla.org/hgcustom/version-control-tools/raw-file/tip/hgext/robustcheckout/__init__.py /usr/local/src/robustcheckout.py
+ADD https://hg.mozilla.org/mozilla-central/raw-file/260e22f03e984e0ced16b6c5ff63201cdef0a1f6/testing/mozharness/external_tools/robustcheckout.py /usr/local/src/robustcheckout.py
 
 # for testing builds (these lines mirror above), copy above artifacts from the downloads dir
 # COPY downloads/node-v8.11.3-linux-x64.tar.gz /builds/worker/Downloads

